src/main/java/core/basesyntax/processing/FruitTransaction.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.processing;
+
+import java.util.List;
+
+public interface FruitTransaction {
```

Comment:
FruitTransaction should be a model with 3 fields:
`private Operation operation; `- it's an enum (BALANCE, SUPPLY, RETURN, PURCHASE)
`private String fruit;`
`private int quantity;`